src/rules/noUnusedVariableRule.ts

```diff
@@ -16,8 +16,13 @@
 import * as Lint from "../lint";
 import * as ts from "typescript";
 
+const OPTION_REACT = "react";
+const OPTION_REACT_MODULES = ["react", "react/addons"];
+const OPTION_REACT_NAMESPACE_IMPORT_NAME = "React";
```

Comment:
Can these two consts not start with `OPTION_`? I think that should be reserved only for actual options which are provided to the rule